scripts/Fret.js

```diff
@@ -17,8 +17,16 @@ function Fret(x, y, note, string) {
 }
 
 Fret.prototype.drawLines = function() {
-  generateShape([frets[125], frets[126], frets[102], frets[77]]);
-  generateShape([frets[3], frets[4], frets[29], frets[28]]);
+  for (var i = 0; i < shapes.length; i++) {
+    var currentShape = shapes[i];
+    var currentShapeArray = [];
+    for (var item in currentShape) {
+      if (currentShape.hasOwnProperty(item)) {
```

Comment:
This seems like a redundant line, we already know the `currentShape` has the `item` because we're looping through it directly.